components/requests/DoctorInformationCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, HStack, Text, Divider, SimpleGrid, Link } from '@chakra-ui/react';
+import Card from '@components/internal/Card';
+
+type doctorInformationProps = {
+  readonly name: string;
+  readonly mspNumber: number;
+  readonly phoneNumber: string;
+  readonly province: string;
+  readonly city: string;
+  readonly address: string;
+  readonly postalCode: string;
+};
+
+export default function DoctorInformationCard(props: doctorInformationProps) {
```

Comment:
Instead of passing every single physician attribute as its own individual prop, we want to pass in the physician object as a whole as the prop. You can find the physician type in `lib/graphql/types.ts`